internal/util/configdiag.go

```diff
@@ -6,6 +6,7 @@ import "github.com/kong/go-database-reconciler/pkg/file"
 type ConfigDump struct {
 	Config file.Content
 	Failed bool
+	Raw    []byte
```

Comment:
In the context of this struct, I would say `Raw` is a raw config dump. Could we make it more elaborate to make sure the meaning is conveyed right (e.g. `RawFailedHTTPResponse`)?

Reply:
[945b105](https://github.com/Kong/kubernetes-ingress-controller/pull/5773/commits/945b1050bb6da7d454c04450289ae1ca34a51554)

Reply:
Hey, I renamed _some_ things, though yeah, IDK why I renamed everything but this earlier.

bc384a74c